metrics/codahale.go

```diff
@@ -111,6 +111,10 @@ func (c *CodaHale) UpdateGauge(key string, v float64) {
 	c.getGauge(key).Update(v)
 }
 
+func (c *CodaHale) UpdateRoutesGauge(v int) {
+	// not implemented
+}
```

Comment:
There is `UpdateGauge` which supports custom key. Check how it is used elsewhere, I think we'd prefer it unless there is a solid reason not to.

Reply:
great point, this makes the implementation even simplier!